src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothAudio.cs

```diff
@@ -105,5 +105,98 @@ public void Disconnect(BluetoothAudioProfileType type)
                 BluetoothErrorFactory.ThrowBluetoothException((int)BluetoothError.NotEnabled);
             }
         }
+
+        /// <summary>
+        /// Opens a SCO(Synchronous Connection Oriented link) to connected remote device, asynchronously.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public void OpenAgSco()
```

Comment:
Ag is Audio Gateway.

Reply:
How about change to OpenAudioGatewaySco?

Reply:
It seems too long because the name of all api should be changed.
How about add the xml document?
"Open a AG(Audio Gateway) SCO(Synchronous Connection Oriented link) to 'connected ..."

Reply:
Ok, Please update description.